my-app/src/Map.js

```diff
@@ -0,0 +1,129 @@
+import { useSearchParams } from 'react-router-dom';
+import React, { useState } from 'react';
+import { GoogleMap, useJsApiLoader , Marker} from "@react-google-maps/api";
+import './css/Map.css';
+
+
+
+
+function Map(){
```

Comment:
Nit: Let's use lowercase for the filename and function names.

Reply:
I don't think I followed camelCase for function name before but I will change this too lol